integration_tests/pyproject.toml

```diff
@@ -26,6 +26,7 @@ cprotobuf = "^0.1.11"
 flaky = "^3.7"
 eth-bloom = "^3.0"
 eth-hash = "^0"
+pyyaml = { git = "https://github.com/yaml/pyyaml.git", branch = "main" }
```

Comment:
https://github.com/yaml/pyyaml/pull/809
`6.0.2rc1` seems to be released soon.

Reply:
wow, 5h ago